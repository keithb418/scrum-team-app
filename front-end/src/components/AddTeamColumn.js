import React from "react";
import { Button } from "react-bootstrap";
import FontAwesome from "react-fontawesome";

const AddTeamColumn = () => {
  return (
    <Button className="add-team-button">
      <span className="content">
        <FontAwesome
          name="plus-circle"
        />
        Add Team Column
      </span>
    </Button>
  );
};

export default AddTeamColumn;
