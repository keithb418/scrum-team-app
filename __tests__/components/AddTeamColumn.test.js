import React from "react";
import { shallow } from "enzyme";
import AddTeamColumn from "../../front-end/src/components/AddTeamColumn";

test('should render the AddTeamColumn component correctly', () => {
  const wrapper = shallow(<AddTeamColumn />);
  expect(wrapper).toMatchSnapshot();
});

