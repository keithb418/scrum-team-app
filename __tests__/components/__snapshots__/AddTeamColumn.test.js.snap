// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the AddTeamColumn component correctly 1`] = `
<div>
  <Link
    replace={false}
    to="/team/add"
  >
    <Button
      active={false}
      block={false}
      bsClass="btn"
      bsStyle="default"
      className="add-team-button"
      disabled={false}
    >
      <span
        className="content"
      >
        <FontAwesome
          name="plus-circle"
        />
        Add Team Column
      </span>
    </Button>
  </Link>
</div>
`;
